flow/scheduling/torque.py

```diff
@@ -141,7 +141,11 @@ def submit(self, script, after=None, pretend=False, hold=False, flags=None, *arg
     def is_present(cls):
         "Return True if it appears that a TORQUE scheduler is available within the environment."
         try:
-            subprocess.check_output(['qsub', '--version'], stderr=subprocess.STDOUT)
+            # some qsub wrappers don't run on no-tty
+            import pty
+            master, slave = pty.openpty()
+            subprocess.Popen(['qsub', '--version'], stdin=slave, stdout=slave, stderr=slave,
+                             shell=True)
```

Comment:
It appears that this function is not reliably raising either an `IOError` or `OSError` to indicate the scheduler is *not* present.

For example, I executed this command on a machine that I know has no Torque scheduling system: 
```
$ python -c "from flow.scheduling import TorqueScheduler; print(TorqueScheduler.is_present())"
True
```
I'm pretty sure that this is also the reason why unit tests are failing.